### PR TITLE
Tbcm 3 create common frontend components

### PR DESCRIPTION
**Pull request by PeterPavlatos**
Create common frontend components

- Button
- Checkbox
- Radiobox
- Table
- TextField
- Label
- Description

I was having some issues with modal. I will fix after. This way other devs can use the components in their tasks.

### CODE REVIEW

File: apps/web/src/components/Checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+// import { OptionType } from '@components';
+// import { Field as FormikField } from 'formik';
```
Comment (karthikktamilmani-freshworks) [inline apps/web/src/components/Checkbox.tsx:2]: Can be removed.

File: apps/web/src/styles/base/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// Primary BC Brand 
+$bc-primary_blue: #003366;
```
Comment (karthikktamilmani-freshworks) [inline apps/web/src/styles/base/_variables.scss:2]: I don't see this being used?

File: package.json [hunk 3]
```diff
@@ -50,11 +50,14 @@
     "wsrun": "5.2.4"
   },
   "dependencies": {
+    "@headlessui/react": "^1.6.6",
```
Comment (karthikktamilmani-freshworks) [inline package.json:53]: Could you lock the versions and remove ^

File: package.json [hunk 4]
```diff
@@ -50,11 +50,14 @@
...
+    "@heroicons/react": "^2.0.10",
```
Comment (karthikktamilmani-freshworks) [inline package.json:54]: Are we using this?

File: package.json [hunk 5]
```diff
@@ -50,11 +50,14 @@
...
     "@types/papaparse": "5.3.2",
     "dayjs": "1.11.0",
     "papaparse": "5.3.1",
     "react-dropzone": "12.0.4",
-    "swr": "1.3.0"
+    "swr": "1.3.0",
+    "tailwindcss": "^3.1.8"
```
Comment (karthikktamilmani-freshworks) [inline package.json:60]: These changes should be added to the package.json available inside `apps/web`

**Comment by karthikktamilmani-freshworks** [comment]
Overall looks good!

File: apps/web/src/styles/base/_variables.scss (hunk 2 above)
Comment (PeterPavlatos) [reply apps/web/src/styles/base/_variables.scss]: Removed this file. Not being used.

File: package.json (hunk 4 above)
Comment (PeterPavlatos) [reply package.json:54]: No, Removed this from package.

File: package.json (hunk 5 above)
Comment (PeterPavlatos) [reply package.json:60]: Removed this from package.

**Review by karthikktamilmani-freshworks — APPROVED** [review]

**Pull request merged**